engine/src/main/java/org/terasology/engine/core/modes/loadProcesses/InitialiseRendering.java

```diff
@@ -0,0 +1,33 @@
+// Copyright 2021 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.engine.core.modes.loadProcesses;
+
+import org.terasology.engine.context.Context;
+import org.terasology.engine.core.modes.SingleStepLoadProcess;
+import org.terasology.engine.core.module.rendering.RenderingModuleRegistry;
+
+public class InitialiseRendering extends SingleStepLoadProcess {
+    private final Context context;
+
+    public InitialiseRendering(Context context) {
+        this.context = context;
+    }
+
+
+    @Override
+    public String getMessage() {
+        return "Initialising Rendering System...";
+    }
+
+    @Override
+    public boolean step() {
+        context.put(RenderingModuleRegistry.class, new RenderingModuleRegistry());
```

Comment:
What makes this better as a LoadProcess instead of a Subsystem?

more explicit ordering of load processes? or we need a different context here than was available for any of the subsystem lifecycle phases?

(and heck it is a lot of lines of code for these two expressions, either way)

Reply:
we need a new rendering system when you restart a world.

Reply:
For my information: Why do we need a new rendering system when restarting a world?

Reply:
its all module drive there really isn't a reason to keep it around as a singleton.

Reply:
Okay

Reply:
yea.. we do this for quite a few loading things.

Reply:
Can we please persist this information in a docstring. Probably the class doc here is a good place to capture this.